### DISCUSSION TITLE
I use phaserExtension, how to use it in VUE?

### DISCUSSION THREAD

**Discussion by xx0618** [enable3d/enable3d#107] [Help / Questions]
As in the question, when I try to put enable3d in Vue, the console prompts Cannot read property'WEBGL' of undefined, how should I set it?
`import { enable3d, Scene3D, Canvas, THREE } from 'enable3d'
import { Phaser } from 'phaser'`

**Comment by vitalspace** [comment]
How are you importing the libs?


Well I had the same problem in react. just put in the index.html file in the head:

```
     <script src = "/ lib / phaser.min.js"> </script>
     <script src = "/ lib / enable3d.phaserExtension.0.21.0.min.js"> </script>
```

**Reply by xx0618** [reply]
Thank you for your answer, you are right, no more questions. But the page is blank and nothing is displayed. If you enter text in <template>, text will appear, but there will be no 3D content. How is this going?

**Reply by xx0618** [reply]
I was wrong, and it still prompted Cannot read property'WEBGL' of undefined, and the page has no content that is blank.

**Comment by yandeu** [comment]
Maybe this helps: https://github.com/enable3d/enable3d-phaser-project-template/blob/master/src/scripts/game.ts

**Reply by xx0618** [reply]
Refer to the link content, there is no prompt `Cannot read property'WEBGL' of undefined`. But the page hasn't displayed the content yet, I'm still studying the other content in this link, thank you for your answer! Happy Chinese New Year!

**Reply by xx0618** [reply]
> 也许这会有所帮助：[https](https://github.com/enable3d/enable3d-phaser-project-template/blob/master/src/scripts/game.ts) : [//github.com/enable3d/enable3d-phaser-project-template/blob/master/src/scripts/game.ts](https://github.com/enable3d/enable3d-phaser-project-template/blob/master/src/scripts/game.ts)

According to this setting, `ammo.wasm.js` reports `Uncaught SyntaxError: Unexpected token'<'`. I changed the ammo folder to several locations and still the same error, I don't know where I set it wrong.

![10224102613](https://user-images.githubusercontent.com/71865768/108938456-103e9280-768b-11eb-932a-4632aa35ce07.png)
![02816](https://user-images.githubusercontent.com/71865768/108938459-12a0ec80-768b-11eb-8fc5-b555e5919ee4.png)

**Reply by yandeu** [reply]
I have just updated the template to the latest version of enable3d. I don't see this error there.

**Reply by yandeu** [reply]
You need to place the 3 ammo files `ammo.js`, `ammo.wasm.js` and `ammo.wasm.wasm` in a public folder. 
`withPhysics()` then receives the absolute path to that folder.

**Reply by xx0618** [reply]
![10224151520](https://user-images.githubusercontent.com/71865768/109123277-9ab6ed00-7784-11eb-8b2e-0669a60f813f.png)
I placed it like this, but this error still occurs. Do I need to set other content?
![10224151520](https://user-images.githubusercontent.com/71865768/109123277-9ab6ed00-7784-11eb-8b2e-0669a60f813f.png)

**Reply by yandeu** [reply]
Can you give me access to your repository?

**Comment by yandeu** [comment]
I haven't tried it, but maybe it helps:
https://phaser.discourse.group/t/vue-phaser-3-webpack-boilerplate/2656
